Add telemetry docs pages for Dynatrace (#5533)
apollographql/router: docs/source/configuration/telemetry/exporters/metrics/dynatrace.mdx

---
title: Dynatrace exporter (via OTLP)
subtitle: Configure the Dynatrace exporter for metrics
description: Configure the Dynatrace exporter for metrics via OpenTelemetry Protocol (OTLP) in the Apollo Router.
---

Enable and configure the [OTLP exporter](./otlp) for metrics in the Apollo Router for use with [Dynatrace](https://dynatrace.com/).

For general tracing configuration, refer to [Router Metrics Configuration](./overview).

## Dynatrace configuration

To configure the router:

- Enable the [OTLP exporter](./otlp#configuration)

> - You must use the `http` protocol as `grpc` is currently not supported by Dynatrace (See: https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/otlp-export)

Reviewer: ```suggestion
-  Set the `protocol` as `http`;  Dynatrace [doesn't currently support](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/otlp-export) `grpc`
```

- Provide your Dynatrace endpoint

> - Provide your API token in the `Authorization` header

Reviewer: ```suggestion
- Provide your Dynatrace API token in the `Authorization` header; the header should start with [`Api-token` and then your Dynatrace token](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/otlp-export#authentication-export-to-activegate)
```
